node/src/BaseClient.ts

```diff
@@ -2062,7 +2062,7 @@ export class BaseClient {
      * console.log(result); // Output: 2 - Indicates that the list has two elements.
      * ```
      */
-    public async lpushx(key: string, elements: string[]): Promise<number> {
+    public async lpushx(key: GlideString, elements: string[]): Promise<number> {
```

Comment:
```suggestion
    public async lpushx(key: GlideString, elements: GlideString[]): Promise<number> {
```
We also need to update the given elements

Reply:
Updated.